Assets/HoloToolkit-Examples/HeadsUpDirectionIndicator/Example.unity

```diff
@@ -0,0 +1,431 @@
+%YAML 1.1
```

Comment:
This should probaly have a more descriptive name as well like "OutOfScreenIndicatorTest" or "HeadsUpDirectionTest" .unity